docs/guides/test-runner/code-coverage/index.md

```diff
@@ -212,6 +212,10 @@ We now have a failing test but still 100% code coverage.
 
 You should, therefore, see code coverage as a tool that only gives you guidance and help on spotting missing tests, rather than a hard guarantee of code quality.
 
+## Ignoring uncovered lines
+
+In more complex applications, it is likely that you will find yourself creating difficult, if not impossible, to test branches of functionality. While this can absolutely be a pointer to logic that you might look into breaking down into more approachable parts, there will be cases where this is not feasible. In this case you may chose to [ignore lines of code](../../../docs/test-runner/writing-tests/code-coverage.md#ignoring-uncovered-lines) while computing code coverage.
```

Comment:
can we somehow include the code snippet that most will need... could be in this sort of thinking
```
In most cases `/* c8 ignore next */` will do if you wanna know more then...
```

Reply:
Good idea. I was struggling with how to make a reference on both pages without it being the same.